### DISCUSSION TITLE
Adding a toggle to `useHeadroom` for updates after the fixedAt position in the headroom has been scrolled past

### DISCUSSION THREAD

**Discussion by ickynavigator** [mantinedev/mantine#3782] [Feature requests]
When making a sticky navbar in the headroom the pinned value returns to true if you scroll back up (without scrolling to the top) instead of staying frozen till the user actually reaches the top of the page

I was able to toggle it by adding a new option that turns off the `isPinned` check

**Comment by rtivital** [comment]
I did not quite get what should be added. Can you explain it, maybe you have a demo?

**Reply by ickynavigator** [reply]
I have created a [demo](https://codesandbox.io/p/sandbox/eager-kate-6j03g0) that you can check out

when I'm at the bottom of the screen (or really any height greater than the `fixedAt` height) the header goes invisible as soon I go up a little bit instead of staying visible till I actually reach the top of the screen

**Reply by rtivital** [reply]
use-headroom is used to detect when user is scrolling down. In your case, you can simply use position: sticky.

**Reply by ickynavigator** [reply]
thanks
I missed that 😅
